components/CreateNewUserAccount/CreateButton.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { Button, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle } from '@mui/material';
+import axios from 'axios';
+
+type Props = {
+  name : string;
+  email : string;
+  role : string;
+  isCompleted : boolean;
+}
+
+const CreateButton: React.FC<Props> = (props) => {
+    const [open, setOpen] = React.useState(false);
+
+    const handleClickOpen = () => {
+      setOpen(true);
+    };
+
+    const handleClose = () => {
+      setOpen(false)
+    };
+
+    const handleSubmit = () => {
+      setOpen(false);
+      axios.post(`${process.env.NEXT_PUBLIC_API_URL}/user`, {
+        name: props.name,
+        email: props.email,
+        role: props.role,
+      })
+      .then(function (response) {
+        console.log(response);
+      })
+      .catch(function (error) {
+        console.log(error);
```

Comment:
can use the notistick snackbars for user feedback when success and error

Reply:
refer to this PR for how to use notistick in the PublishButton file: https://github.com/dsc-project-mr/mr-fe-internal/pull/44/files#diff-e091c326cab0be9547c7d7faea684d57dc80678afe2a870e62f15401135406d1